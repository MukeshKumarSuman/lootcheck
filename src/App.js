import React, { Component } from 'react';
import './App.css';
import Wallet from './components/Wallet';
import Loot from './components/Loot';

class App extends Component {
 render() {
  return (
    <div className="App">
      <h1>Loot Check</h1>
      <hr />
      <Wallet />
      <Loot />
    </div>
  );
 }
}
export default App;
