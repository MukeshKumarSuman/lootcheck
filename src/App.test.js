import React from 'react';
import { shallow } from 'enzyme';
import {createSerializer} from 'enzyme-to-json';
import App from './App';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));
// My Unit test
describe('App module', () => {

  const app = shallow(<App />);

  test('renders properly', () => {
    expect(app).toMatchSnapshot();
  });

  test('contains the connected wallet component', () => {
    //console.log(app.debug());
    expect(app.find('Connect(Wallet)').exists()).toBe(true);
  });

  test('contains the connected loot component', () => {
    //console.log(app.debug());
    expect(app.find('Connect(Loot)').exists()).toBe(true);
  });

});
