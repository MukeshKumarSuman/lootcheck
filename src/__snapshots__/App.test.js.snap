// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App module renders properly 1`] = `
<div
  className="App"
>
  <h1>
    Loot Check
  </h1>
  <hr />
  <Connect(Wallet) />
  <Connect(Loot) />
</div>
`;
